packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts

```diff
@@ -86,7 +89,9 @@ describe("FtpMvsApi", () => {
     });
 
     it("should upload content to dataset.", async () => {
-        const localFile = "/tmp/testds1.txt";
+        const localFile = tmp.tmpNameSync({ tmpdir: "/tmp" });
```

Comment:
How about other cases using `/tmp/testds1.txt`?

Reply:
Thanks @std4lqi , I updated it.